components/service/glean/src/test/java/mozilla/components/service/glean/scheduler/MetricsPingSchedulerTest.kt

```diff
@@ -440,6 +444,34 @@ class MetricsPingSchedulerTest {
         assertTrue(getWorkerStatus(MetricsPingWorker.TAG).isEnqueued)
     }
 
+    @Test
+    fun `startupCheck happens when returning from background when Glean is already initialized`() {
+        // For faking calling the lifecycle observer.
+        val lifecycleRegistry = LifecycleRegistry(mock(LifecycleOwner::class.java))
+        val gleanLifecycleObserver = GleanLifecycleObserver()
+        lifecycleRegistry.addObserver(gleanLifecycleObserver)
+
+        // Set the last sent date to yesterday.
```

Comment:
nit: is this the wrong comment?

Reply:
Yes it is...  pesky things, I thought I put out a trap for them. ;)